hppc/src/main/templates/com/carrotsearch/hppc/KTypeHashSet.java

```diff
@@ -735,6 +735,27 @@ public void indexInsert(int index, KType key) {
     }
   }
 
+  /**
+   * Removes a key.
```

Comment:
I think it would be safer to say "removes a key index previously acquired from {{@@link #indexOf}}?

Reply:
Right